liberty-maven-plugin/src/it/tests/appsdirectory-set-appz-notconfigured-it/src/main/resources/server.xml

```diff
@@ -0,0 +1,6 @@
+<server description="default server">
+    <featureManager>
+        <feature>jsp-2.3</feature>
+        <webApplication id="${artifactId}" location="${artifactId}.war" name="${artifactId}"/>
```

Comment:
The test name implies that `webApplication` nor `application` are configured.  I think this configuration is being ignored since it is in the scope of `featureManager`. Since the point here is to not configure the application and  make sure that app is installed to dropins, remove this application configuration.